persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/util/CacheMapUtil.java

```diff
@@ -67,10 +67,10 @@
     private static final int REPOSITORY_RESULT_OFFSET = 0; // retrieve all entries (do not skip any)
 
     // maps used to cache required relationships - in all maps stable ids are key
-    private Map<String, MolecularProfile> molecularProfileCache;
-    private Map<String, SampleList> sampleListCache;
-    private Map<String, CancerStudy> cancerStudyCache;
-    private Map<String, String> genericAssayStableIdToMolecularProfileIdCache;
+    private static Map<String, MolecularProfile> molecularProfileCache;
```

Comment:
I know why you made this change, but I don't think anyone else does. Can you add a comment here explaining why these need to be static?

Reply:
Done.